### DISCUSSION TITLE
cannot remove VAT/GST id

### DISCUSSION THREAD

**Discussion by aetonsi** [community/community#67218] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

hi,
whenever i try to remove my "VAT/GST ID" from the [billing/payment_information](https://github.com/settings/billing/payment_information) settings page, it shows up again unaffected.
i think this is a bug.

thanks
- aetonsi

**Comment by artur99** [comment]
Had the same bug as well, I just went and replaced the field's text with `-`.

**Comment by aetonsi** [comment]
personally the bug has been solved, i did nothing to try to solve it recently, but one day the field was empty.
i'll leave this discussion open for other people

**Comment by peter711** [comment]
Hello,
I have the same issue. Cannot remove VAT/GST ID even when I put `-` as a value in input it still stays as it was.

**Comment by swiknaba** [comment]
This bug still exists! :(add a non-breaking space to overwrite it)

**Comment by Hanashiko** [comment]
Hi there,

If your VAT/GST ID reappears after attempting to remove it via the Billing > Payment Information settings page, and the change doesn’t persist, it's very likely a bug in the UI or backend validation handling.

Suggested Next Steps:
1. Try an alternate browser or incognito mode to rule out any caching or extension-related issues.
2. Check for any visible error messages in the browser developer console (e.g., network errors, validation issues).
3. Submit a support ticket to GitHub directly: Since this relates to billing and account data, the best course of action would be to [open a ticket through GitHub Support](https://support.github.com/contact) — they can investigate securely and make any necessary corrections on your behalf.
